torch/csrc/distributed/rpc/init.cpp

```diff
@@ -195,6 +195,9 @@ PyObject* rpc_init(PyObject* /* unused */) {
           .def(
               py::pickle(
                   [](const PyRRef& self) {
+                    TORCH_CHECK(
+                        false,
+                        "Can not pickle rref in python pickler, rref can only be pickled in _InternalRPCPickler");
```

Comment:
Since this will be a user message, does it make sense if we don't mention `_InternalRPCPickler` and only say "RRef can only be pickled when using RPC".

Reply:
sounds good